server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/model/message/view/MessageView.scala

```diff
@@ -0,0 +1,25 @@
+/** **************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ * *
+ * http://www.apache.org/licenses/LICENSE-2.0                 *
+ * *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ * ************************************************************** */
+package org.apache.james.jmap.model.message.view
+
+import org.apache.james.mailbox.model.MessageId
+
+trait MessageView {
+  def getId: MessageId
```

Comment:
IMO you should not introduce this class as part of this work but rather add a Keyword property in the existing Email class